WooCommerce/Classes/POS/Presentation/TotalsView.swift

```diff
@@ -267,15 +267,31 @@ private extension TotalsView {
             topPadding: 40,
             bottomPadding: 40
         )
+
+        static let topAligned = CardReaderViewLayout(
```

Comment:
I admit seeing a non-centered payment error view looked weird to me at first 😀.

Figma comment mentioned that it was done to make sure that there's always space for "No internet connection" banner.

Either way, happy that `CardReaderViewLayout` has proven to be useful in one more case.

Reply:
Yeah I was dreading this one when I saw how the code was written, then really happy to come across the Layout ❤️